packages/client-sdk-web/src/config/configurations.ts

```diff
@@ -0,0 +1,152 @@
+import {CacheConfiguration} from './configuration';
+import {
+  DefaultMomentoLoggerFactory,
+  MomentoLoggerFactory,
+} from '@gomomento/sdk-core';
+import {
+  GrpcConfiguration,
+  StaticGrpcConfiguration,
+  StaticTransportStrategy,
+  TransportStrategy,
+} from './transport';
+
+const defaultLoggerFactory: MomentoLoggerFactory =
+  new DefaultMomentoLoggerFactory();
+
+/**
+ * Laptop config provides defaults suitable for a medium-to-high-latency dev environment.  Permissive timeouts, retries, and
+ * relaxed latency and throughput targets.
+ * @export
+ * @class Laptop
+ */
+export class Laptop extends CacheConfiguration {
```

Comment:
For web, the use cases are very different from the other SDKs.  It's probably okay to keep a Laptop config for developers to use when testing.  But we probably don't need an InRegion config, at least not yet.  What we should add instead is a Browser config.  and the timeouts for the browser config should be lenient, probably the same as Laptop.

cc: @eaddingtonwhite would like your opinion.

Reply:
Makes sense. Do we care about low latency or should we just have Laptop and Browser?

Reply:
let's just start with laptop and browser imo